module-gui/gui/core/RawFont.cpp

```diff
@@ -1,78 +1,75 @@
-// Copyright (c) 2017-2022, Mudita Sp. z.o.o. All rights reserved.
+// Copyright (c) 2017-2023, Mudita Sp. z.o.o. All rights reserved.
 // For licensing, see https://github.com/mudita/MuditaOS/LICENSE.md
 
 #include "RawFont.hpp"
-#include "Common.hpp"        // for Status, Status::GUI_SUCCESS, Status::GU...
-#include "Context.hpp"       // for Context
-#include "DrawCommand.hpp"   // for DrawRectangle, DrawCommand (ptr only)
-#include "FontKerning.hpp"   // for FontKerning
-#include "Renderer.hpp"      // for Renderer
-#include "TextConstants.hpp" // for newline
-#include <log/log.hpp>       // for LOG_ERROR
-#include "utf8/UTF8.hpp"     // for UTF8
-#include <cstring>           // for memcpy
-#include <utility>           // for pair
+#include "Common.hpp"
+#include "Context.hpp"
+#include "renderers/LineRenderer.hpp"
+#include "renderers/RectangleRenderer.hpp"
+#include "TextConstants.hpp"
+#include <log/log.hpp>
+#include "utf8/UTF8.hpp"
+#include <cstring>
+#include <utility>
 
 namespace gui
 {
-
     RawFont::~RawFont()
     {
-        if (fallback_font != nullptr) {
-            fallback_font = nullptr;
-        }
+        fallback_font = nullptr;
     }
 
     gui::Status RawFont::load(uint8_t *data)
     {
-        uint32_t offset = 0;
+        std::uint32_t offset = 0;
```

Comment:
I think that in the future such cleanup changes (also e.g. adding `const`) could be made in a separate commit so that the PR is much more readable

Reply:
You're absolutely right, I'll follow that convention next time!